controllers/members.js

```diff
@@ -49,7 +53,37 @@ const getIdleMembers = async (req, res) => {
   }
 }
 
+const purgeMembersCache = async (req, res) => {
+  try {
+    const { username } = req.userData?.username
```

Comment:
JSDoc missing, please add

Reply:
Is there anything wrong with `const { username } = req.userData` ? Just asking 🤔

Reply:
Nope we get the username of the currently logged in by getting the data with the help of the cookie